docs/html/installation.md

```diff
@@ -0,0 +1,78 @@
+# Installation
+
+Usually, pip is automatically installed if you are:
+
+- working in a
+  {ref}`virtual environment <pypug:Creating and using Virtual Environments>`
+- using Python downloaded from [python.org](https://www.python.org)
+- using Python that has not been modified by a redistributor to remove
+  {mod}`ensurepip`
+
+## Supported Methods
+
+If your Python environment does not have pip installed, there are 2 mechanisms
+to install pip supported directly by pip's maintainers:
+
+- [`ensurepip`](#using-ensurepip)
+- [`get-pip.py`](#using-get-pip-py)
+
+### `ensurepip`
+
+Python comes with an {mod}`ensurepip` module[^python], which can install pip in
+a Python environment.
+
+```{pip-cli}
+$ python -m ensurepip --upgrade
+```
+
+More details about how {mod}`ensurepip` works and how it can be used, is
+available in the standard library documentation.
+
+### `get-pip.py`
```

Comment:
@pradyunsg looks like this title has lost its label causing https://github.com/pypa/packaging.python.org/pull/898/checks?check_run_id=2655734156#step:6:34 to fail.

Reply:
Hm... I think this refactoring removed a lot of labels: https://webknjaz.github.io/intersphinx-untangled/pip.pypa.io/

Reply:
I wonder if it's time to enable `autolablel`.

Reply:
> @pradyunsg looks like this title has lost its label causing https://github.com/pypa/packaging.python.org/pull/898/checks?check_run_id=2655734156#step:6:34

Ah, nice catch.

We should probably update that to point to https://github.com/pypa/get-pip/#readme instead, since that's the source of truth for information about get-pip now.

> I wonder if it's time to enable `autolabel`.

I don't want every heading to be an "intersphinx contract" that this documentation needs to be mindful of when making changes.

> Hm... I think this refactoring removed a lot of labels:

That's correct. Why is that a problem?

Reply:
> That's correct. Is that a problem somehow?

My understanding is that they used to be exposed via intersphinx and now they are not. So this has probably broken some sphinx sites.

Reply:
> I don't want every heading to be an "intersphinx contract" that this documentation needs to be mindful of when making changes.

I agree with this. I don't think that our previous documentation was well-enough structured that we want to be blocked from improving it just because some sites linked to individual sections. (I don't think that docs are *ever* so perfect that not being able to reorganise them is reasonable, actually...)

Reply:
> I agree with this. I don't think that our previous documentation was well-enough structured that we want to be blocked from improving it just because some sites linked to individual sections. (I don't think that docs are _ever_ so perfect that not being able to reorganise them is reasonable, actually...)

I believe that Pradyun's comment was not about the old titles but about enabling automatic label creation based on title names. Sphinx has a built-in extension called `autolabel` for this. With this, there wouldn't be any need to create manual labels before titles.